Makefile

```diff
@@ -97,7 +97,7 @@ vet:
 
 .PHONY: generate
 generate:
-	GO111MODULE=on $(GO) generate $(PACKAGES)
+	GO111MODULE=on $(GO) generate -mod=vendor $(PACKAGES)
```

Comment:
if we add this, then we should make minimum supported version of go 1.11

Reply:
I think we have, since `make build` has already use `-mod=vendor` parameters.

Reply:
Can you update here: https://github.com/go-gitea/gitea/blob/master/.drone.yml#L66 and https://github.com/go-gitea/gitea/blob/master/docs/content/doc/installation/from-source.en-us.md to say 1.11 is minimum

Reply:
also on https://github.com/go-gitea/gitea/blob/master/docs/content/doc/advanced/hacking-on-gitea.en-us.md

Reply:
done.